src/main/java/mate/academy/intro/dto/CreateBookRequestDto.java

```diff
@@ -1,14 +1,23 @@
 package mate.academy.intro.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
+    @NotNull
     private String title;
+    @NotNull
     private String author;
+    @NotNull
     private String isbn;
+    @NotNull
+    @Min(0)
     private BigDecimal price;
+    @NotNull
     private String description;
+    @NotNull
     private String coverImage;
```

Comment:
there are no requirements in task description for these fields
```suggestion
    private String description;
    private String coverImage;
```